index.js

```diff
@@ -32,7 +33,7 @@ CacheBreaker.prototype.cacheBreakPath = function(base, resource) {
   var extname = path.extname(resource);
   var basename = path.basename(resource, extname);
 
-  return path.join(dirname, basename + '.' + cs.substring(0, 10) + extname);
+  return url.resolve(dirname + '/', basename + '.' + cs.substring(0, 10) + extname);
```

Comment:
Hmm, I'm cache breaking like this:
```
        <link rel="stylesheet" type="text/css" media="screen" href="{{cache-break:css/toastr.css}}">
```

And then I'm using `gulp-hasher` to add the SHA1 on the css file that I'm running through gulp as part of the copy from my `app` folder to my `dist` folder. I could definitely be missing something in what I should be doing.